app/components/pages/SubmissionWizard/steps/Files/index.js

```diff
@@ -95,12 +123,22 @@ const FilesPageContainer = ({
   uploadData = {},
   uploadLoading,
 }) => (
-  <Mutation mutation={UPLOAD_MUTATION}>
+  <Mutation mutation={UPLOAD_MANUSCRIPT_MUTATION}>
     {(uploadFile, { loading, error: uploadError }) => {
       const fieldName = 'files'
```

Comment:
Not sure if `fieldName` here is very descriptive ... I'd use `const submissionFiles = values['files]`